src/constants/routes.ts

```diff
@@ -0,0 +1,5 @@
+export const ROUTES = {
+  LOGIN: "Login",
+  SIGNUP: "SignUp",
+  HOME: "Home",
+} as const;
```

Comment:
mention the reason for `as const`

Reply:
so that the properties are taken as its routes only
without this it would be taken as string and it will be unable to find the route